apps/console/src/features/organization-discovery/components/edit-organization-discovery-domains.tsx

```diff
@@ -32,7 +32,7 @@ import {
 } from "@wso2is/core/models";
 import { addAlert } from "@wso2is/core/store";
 import { FinalForm, FinalFormField, FormRenderProps, TextFieldAdapter } from "@wso2is/form";
-import { ContentLoader, EmphasizedSegment, Hint, PrimaryButton } from "@wso2is/react-components";
+import { ContentLoader, EmphasizedSegment, Hint, Message, PrimaryButton } from "@wso2is/react-components";
```

Comment:
Let's check if we can use oxygen UI components

Reply:
Fixed with 42d9121fd56619d01fc68ce4f511a68867512d54